components/admin_console/audits/audits.tsx

```diff
@@ -11,52 +10,47 @@ import LoadingScreen from 'components/loading_screen';
 
 import ReloadIcon from 'components/widgets/icons/fa_reload_icon';
 
-export default class Audits extends React.PureComponent {
-    static propTypes = {
-        isLicensed: PropTypes.bool.isRequired,
+type Props = {
+    isLicensed: boolean;
+    audits: Array<any>;
+    actions: {
+        getAudits: () => any;
```

Comment:
I believe this has to be `() => ActionFunc`

Reply:
I tried switching this to ActionFunc but get the error "Property 'then' does not exist on type 'ActionFunc'"